Marlin/Configuration_adv.h

```diff
@@ -1235,6 +1235,19 @@
    */
   //#define SDCARD_CONNECTION LCD
 
+  //#define SDCARD_READONLY   // Disable write support for SD cards (saves ~2192 PROGMEM bytes)
+  #if ENABLED(SDCARD_READONLY)
```

Comment:
I think it's better to put these in sanity include files

Reply:
Didn't knew that was a thing.  Re-pushed with the checks in `SanityCheck.h`